Tests/ShipTests.cs

```diff
@@ -579,5 +579,30 @@ public void TestShipJumpedEvent()
             // Test the result to verify that the distance is calculated relative to the jump coordinates
             Assert.AreEqual(21996.3M, shipMonitor.GetShip(9999)?.distance);
         }
+
+        [TestMethod]
+        public void TestShipCommanderContinuedEventInSRV()
+        {
+            // Set up our `Ship monitor` private object
+            ShipMonitor shipMonitor = new ShipMonitor();
+            PrivateObject privateObject = new PrivateObject(shipMonitor);
+            privateObject.SetFieldOrProperty("updatedAt", DateTime.MinValue);
+
+            // Set up our `currentShipId` property with a value of "9999"
+            privateObject.SetFieldOrProperty("currentShipId", 9999);
+
+            // Set up our event (which reports that we are in an SRV with a ship id of "9998")
+            string line = @"{ ""timestamp"":""2020-07-20T15:40:45Z"", ""event"":""LoadGame"", ""FID"":""F0000000"", ""Commander"":""TestCommander"", ""Horizons"":true, ""Ship"":""TestBuggy"", ""Ship_Localised"":""SRV Scarab"", ""ShipID"":9998, ""ShipName"":"""", ""ShipIdent"":"""", ""FuelLevel"":0.000000, ""FuelCapacity"":0.000000, ""GameMode"":""Group"", ""Group"":""Children of Raxxla"", ""Credits"":5065687467, ""Loan"":0 }";
```

Comment:
IIUC, this test was added to check that the code recognizes the string `SRV`.
Should we also have a test to check for `TestBuggy`?

Reply:
It actually is constructed using "TestBuggy" as the ship (matching what we see in-game). The name isn't transformed from "TestBuggy" to "SRV"  until the .ctor for the `Commander continued` event.

The solution is designed to work equally well whether "TestBuggy" or "SRV" is passed from the `Commander continued` event.

Reply:
I see.
Is there any way that we can test both cases?
Which was my original point but I worded it soo poorly that it got lost... :-D

Reply:
Test updated.